pkg/cluster/bootstrap.go

```diff
@@ -53,6 +53,8 @@ func (c *Cluster) Bootstrap(ctx context.Context, snapshot bool) error {
 				if err := c.httpBootstrap(ctx); err == nil {
 					logrus.Info("Successfully reconciled with datastore")
 					return nil
+				} else {
+					logrus.Warn("Attempt to httpBootstrap failed: ", err.Error())
```

Comment:
nit: make the failure message consistent with the success message?
```suggestion
					logrus.Warnf("Unable to reconcile with datastore: %v", err)
```

Reply:
Updated